api/core/v1alpha2/vdcondition/condition.go

```diff
@@ -28,8 +28,8 @@ const (
 	DatasourceReadyType Type = "DatasourceReady"
 	// ReadyType indicates whether the import process succeeded and the `VirtualDisk` is ready for use.
 	ReadyType Type = "Ready"
-	// ResizedType indicates whether the disk resizing operation is completed.
-	ResizedType Type = "Resized"
+	// ResizingType indicates whether the disk resizing operation is completed.
+	ResizingType Type = "Resizing"
```

Comment:
Let's get rid of the ResizingNotRequested and Resized reasons, it's legacy now, no one needs it.